fcli-core/fcli-common/src/main/java/com/fortify/cli/common/output/cli/mixin/OutputHelperMixins.java

```diff
@@ -44,172 +44,176 @@
     }
-    
+
     public static class Generate extends DetailsNoQuery {
         public static final String CMD_NAME = "generate";
     }
-    
+
     public static class GenerateConfig extends TableNoQuery {
         public static final String CMD_NAME = "generate-config";
     }
-    
+
     public static class Status extends TableNoQuery {
         public static final String CMD_NAME = "status";
     }
-    
-    
+
+
     public static class Set extends TableNoQuery {
         public static final String CMD_NAME = "set";
    }
-    
-    
+
+
     public static class Update extends TableNoQuery {
         public static final String CMD_NAME = "update";
     }
-    
-    
+
+
     public static class Enable extends TableNoQuery {
         public static final String CMD_NAME = "enable";
     }
-    
-    
+
+
     public static class Disable extends TableNoQuery {
         public static final String CMD_NAME = "disable";
     }
-    
-    
+
+
     public static class Start extends TableNoQuery {
         public static final String CMD_NAME = "start";
     }
-    
-    
+
+
     public static class Pause extends TableNoQuery {
         public static final String CMD_NAME = "pause";
     }
-    
-    
+
+
     public static class Resume extends TableNoQuery {
         public static final String CMD_NAME = "resume";
     }
-    
-    
+
+
     public static class Cancel extends TableNoQuery {
         public static final String CMD_NAME = "cancel";
     }
 
-    
+
     public static class Upload extends TableNoQuery {
         public static final String CMD_NAME = "upload";
     }
-    
-    
+
+
     public static class Download extends TableNoQuery {
         public static final String CMD_NAME = "download";
     }
-    
-    
+
+
     public static class Install extends TableNoQuery {
         public static final String CMD_NAME = "install";
     }
-    
-    
+
+
     public static class Uninstall extends TableNoQuery {
         public static final String CMD_NAME = "uninstall";
     }
 
-    
+
     public static class Import extends TableNoQuery {
         public static final String CMD_NAME = "import";
     }
 
-    
+
     public static class Export extends TableNoQuery {
         public static final String CMD_NAME = "export";
     }
 
-    
+
     public static class Setup extends TableNoQuery {
         public static final String CMD_NAME = "setup";
     }
-    
+
     public static class WaitFor extends TableNoQuery {
         public static final String CMD_NAME = "wait-for";
     }
-    
+
     public static class Login extends TableNoQuery {
         public static final String CMD_NAME = "login";
     }
-    
-    
+
+
     public static class Logout extends TableNoQuery {
         public static final String CMD_NAME = "logout";
     }
-    
+
     public static class RestCall extends DetailsWithQuery {
         public static final String CMD_NAME = "call";
     }
+
+    public static class LookupItems extends TableWithQuery {
```

Comment:
I can imagine that if we ever want to have a lookup command in another module, it could need a different superclass like `DetailsNoQuery`. Also, for a generic command like this, I'd use a name that better matches the actual command name, so `Lookup` would be better.

Given the above, I'd like to suggest to either:
* Keep this class in the generic `OutputHelperMixins` class, but rename to `LookupList` (so we can potentially also have a `LookupDetails` class with same `CMD_NAME` but different superclass)
* Move this class to `FoDOutputHelperMixins`, using any name you like ;)

Reply:
Will do, don't know why I placed it here!